test/box-luatest/downgrade_test.lua

```diff
@@ -973,3 +973,40 @@ g.test_downgrade_global_names = function(cg)
         t.assert_equals(#box.space._cluster:format(), 2)
     end)
 end
+
+-----------------------------
+-- Check downgrade from 3.1.0
+-----------------------------
+
+g.test_downgrade_func_trigger = function(cg)
+    cg.server:exec(function()
+        local helper = require('test.box-luatest.downgrade_helper')
+        local trigger = require('trigger')
+        local _func = box.space._func
+        local prev_version = helper.prev_version(helper.app_version('3.0.0'))
+        local lua_code = 'function(a, b) return a + b end'
+        t.assert_equals(trigger.info('test'), {})
+        box.schema.func.create('trigger_func',
+            {body = lua_code, trigger = 'test'})
+        local errmsg = 'Function trigger_func is registered as ' ..
+            'event trigger. It is supported starting from version 3.1.0'
+        t.assert_equals(box.schema.downgrade_issues(prev_version), {errmsg})
```

Comment:
Let's 
- check that we set correctly from version in downgrade. That is there is no with issues downgrading to `helper.app_version('3.1.0')` when persistent trigger is present (see `test_downgrade_schema_max_id` for example).
- Nit. check that `downgrade_issues` works with correctly with several issues.

Reply:
Done.